pyproject.toml

```diff
@@ -3,5 +3,6 @@ requires = [
     "param >=1.9.0",
     "pyct >=0.4.4",
     "bokeh >=2.4.0,<2.5",
-    "pyviz_comms >=0.6.0"
+    "pyviz_comms >=0.6.0",
+    "setuptools"
```

Comment:
`setuptools` would also have to be included in the `build` extra group declared in `setup.py`. As I understand it, the  build dependencies declared in `pyproject.toml` are picked up by pip while the build dependencies declared in `setup.py` are to be picked up by pyctdev when you run `doit develop_install`.

Reply:
Had a quick look at `pyctdev`'s code and actually it seems that it gets the build dependencies from the pyproject.toml file. So the `build` group extra may just be legacy. It may be worth confirming that.